specification/devcenter/DevCenter/DevBox/routes.tsp

```diff
@@ -6,22 +6,75 @@ import "../shared/routes.tsp";
 using Azure.Core;
 using TypeSpec.Rest;
 using TypeSpec.Http;
+using TypeSpec.Versioning;
 
 namespace DevCenterService;
 
 interface DevBoxes {
-  @doc("Lists available pools")
+  @doc("Lists available pools.")
   listPools is StandardResourceOperations.ResourceList<Pool>;
 
-  @doc("Gets a pool")
+  @doc("Gets a pool.")
   getPool is StandardResourceOperations.ResourceRead<Pool>;
 
-  @doc("Lists available schedules for a pool.")
-  listSchedulesByPool is StandardResourceOperations.ResourceList<Schedule>;
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Need of reuse ScheduleListResult"
+  // Alternate route to return same list response of Schedules
+  @doc("Lists all schedules within a pool that are configured by your project administrator.")
+  @route("/projects/{projectName}/pools/{poolName}/schedules")
+  @get
+  listSchedulesByPool is Azure.Core.Foundations.Operation<
+    {
+      @doc("The DevCenter Project upon which to execute operations.")
+      @maxLength(63)
+      @minLength(3)
+      @pattern("^[a-zA-Z0-9][a-zA-Z0-9-_.]{2,62}$")
+      @path
+      projectName: string;
+
+      @doc("The DevCenter Project upon which to execute operations.")
+      @maxLength(63)
+      @minLength(3)
+      @pattern("^[a-zA-Z0-9][a-zA-Z0-9-_.]{2,62}$")
+      @path
+      poolName: string;
+    },
+    ScheduleListResult
+  >;
 
   @doc("Gets a schedule.")
   getScheduleByPool is StandardResourceOperations.ResourceRead<Schedule>;
 
+  #suppress "@azure-tools/typespec-azure-core/use-standard-operations" "Need of reuse ScheduleListResult"
+  // Alternate route to return same list response of Schedules
+  @doc("Lists all schedules within a project that are configured by your project administrator.")
+  @route("/projects/{projectName}/schedules")
+  @added(APIVersions.v2024_02_01)
+  listSchedulesByProject is Azure.Core.Foundations.Operation<
+    {
+      /**
+       * The maximum number of resources to return from the operation. Example: 'top=10'.
+       */
+      @query("top")
+      top?: int32;
+
+      /**
+       * An OData filter clause to apply to the operation.
+       */
+      @query("filter")
+      filter?: string;
```

Comment:
this was removed because it's not supported by our backend for 2023-04-01. We need to check if that is still the case or if we should indeed have this for 2024-02-01

Reply:
Interesting, you're right in it's not in the generated swagger for 2023-04-01 (`devcenter.json`), but it is in the swagger checked into our repo `src/sdk/specification/devcenter/data-plane/Microsoft.DevCenter/stable/2023-04-01/devbox.json`

Do you know who I can ask to see if this is supported or not?

Reply:
it's checked in our repo and it was published like that for 2023-04-01 swagger, but if you send a request to our service with the filter parameter we would just ignore that (because or backend actually does not support it, even though we have added this to the swagger initially). You should ask to Yiming and Chris how is that going

Reply:
Spoke with Yiming, confirmed this is still not supported. I'll remove it.